docker/accumulo/Dockerfile

```diff
@@ -47,13 +52,13 @@ ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
 COPY ./files/ .
```

Comment:
The Accumulo Dockerfile allows users to supply their own builds of Accumulo, ZooKeeper and Hadoop. Would it be possible to also add this to the hdfs Dockerfile / `build-hadoop.sh`? This way if I supply my own builds, no downloads have to happen at all?
Would be greatly appreciated if this would work as I have had a pain getting anything downloading this week.

Reply:
Never mind, it seems that functionality exists already https://github.com/gchq/gaffer-docker/blob/e040f871024ef3d01abc05b9e6990949f25f401a/docker/hdfs/files/build-hadoop.sh#L25